h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -162,22 +162,22 @@ private void setDefaultMetricAndDirection(Model m) {
     if (m._output.isBinomialClassifier()) { //Binomial
       metrics = new String[]{"logloss", "mean_per_class_error", "rmse", "mse"};
```

Comment:
Ideally use the enum in those string arrays as well

Reply:
good catch